discretize/base/base_tensor_mesh.py

```diff
@@ -22,20 +36,37 @@ class BaseTensorMesh(BaseMesh):
     :meth:`discretize.TensorMesh` class to create a cartesian tensor mesh.
 
     """
-    _meshType = 'BASETENSOR'
+
+    _meshType = "BASETENSOR"
+    _aliases = {
+        **BaseMesh._aliases,
+        **{
+            "gridCC": "grid_cell_centers",
+            "gridN": "grid_nodes",
+            "gridFx": "grid_faces_x",
+            "gridFy": "grid_faces_y",
+            "gridFz": "grid_faces_z",
+            "gridEx": "grid_edges_x",
+            "gridEy": "grid_edges_y",
+            "gridEz": "grid_edges_z",
+        },
```

Comment:
I know we chatted a bit about this at the meeting, but just wanted to flag this as a potential item to revisit: do we want to prepend everything with `grid` or just have `cell_centers`, `nodes`, etc as the properties. As @jcapriot, `grid` implies "a framework of spaced bars that are parallel to or cross each other"

![image](https://user-images.githubusercontent.com/6361812/98836739-48350900-23ff-11eb-9aa7-4985b4c71622.png)